src/thx/DateTime.hx

```diff
@@ -145,9 +145,12 @@ match `end`. No interpolation is made.
     return days;
   }
 
-  inline public static function compare(a : DateTime, b : DateTime)
+  inline public static function compare(a : DateTime, b : DateTime): Int
     return a.compareTo(b);
 
+  inline public static function ord(): Ord<DateTime> 
```

Comment:
nothing wrong with this but wouldn't it be a little more consistent if this was an `Ord` instead of returning one?